model/src/main/scala/hmda/model/census/CBSAMetroMicroLookup.scala

```diff
@@ -0,0 +1,54 @@
+package hmda.model.census
+
+import java.io.File
+
+import scala.io.Source
+
+object CBSAMetroMicroLookup {
+  val values: Seq[CBSAMetroMicro] = {
+    val lines = Source.fromFile(new File("model/src/main/resources/tl_2013_us_cbsa.csv")).getLines
+    lines.map { line =>
+      val values = line.split('|').map(_.trim)
+      val CSAFP = values(0)
+      val CBSAFP = values(1)
+      val GEOIOD = values(2)
+      val name = values(3)
+      val nameLSAD = values(4)
+      val LSAD = values(5)
+      val MEMI = values(6).toInt
+      val MTFCC = values(7)
+      val ALAND = values(8)
+      val AWATER = values(9)
+      val INTPTLAT = values(10)
+      val INTPTLON = values(11)
+
+      CBSAMetroMicro(
+        CBSAFP,
+        GEOIOD,
+        name,
+        nameLSAD,
+        LSAD,
+        MEMI,
+        MTFCC,
+        ALAND,
+        AWATER,
+        INTPTLAT,
+        INTPTLON
+      )
+    }.toSeq
+  }
+}
+
+case class CBSAMetroMicro(
+  CBSAFP: String,
+  GEOIOD: String,
```

Comment:
Is this a value that is (potentially) the same sort of thing as a `CBSATract.geoIdMsa`? Because if so, then I suspect it is mis-spelled (extra `O`), and something like `geoId` would be better.

And probably in general, we want to use a camel case naming convention more like in `CBSATract`.  (And we may want to start using an explicit style guide, but that's outside the scope of this PR.)